lib/tasks/counter_metrics.rake

```diff
@@ -9,13 +9,36 @@ namespace :counter_metrics do
     ImportCounterMetrics.import_requests(args.csv_path)
   end
 
-  # bundle exec rake counter_metrics:import_requests['pittir.hykucommons.org','spec/fixtures/csv/pittir-downloads.csv']
   desc 'import historical counter investigations'
+  # bundle exec rake counter_metrics:import_requests['pittir.hykucommons.org','spec/fixtures/csv/pittir-downloads.csv']
   task 'import_investigations', [:tenant_cname, :csv_path] => [:environment] do |_cmd, args|
     raise ArgumentError, 'A tenant cname is required: `rake counter_metrics:import_investigations[tenant_cname]`' if args.tenant_cname.blank?
     raise ArgumentError, "Tenant not found: #{args.tenant_cname}. Are you sure this is the correct tenant cname?" unless Account.where(cname: args.tenant_cname).first
     AccountElevator.switch!(args.tenant_cname)
     puts "Importing historical investigations for #{args.tenant_cname}"
     ImportCounterMetrics.import_investigations(args.csv_path)
   end
+
+  # You can optionally pass work IDs to have hyrax counter metrics created for specific works.
+  # Or, you can pass a limit for the number of CounterMetric entries you would like to create. currently they are randomly created.
+  # Example for pitt tenant in dev without ids:
```

Comment:
```suggestion
  #
  # Example for pitt tenant in dev without ids:
```

Reply:
The three suggestions give some breathing room to the examples; helps scalability of code.